frontend/tests/frontend.spec.js

```diff
@@ -0,0 +1,160 @@
+const { test, expect, _electron: electron } = require("@playwright/test");
+
+let electronApp;
+let window;
+
+test.beforeAll(async () => {
+  // Launch app
+  electronApp = await electron.launch({ args: ["."] });
+
+  // Wait for the first BrowserWindow to open and return its Page object
+  window = await electronApp.firstWindow();
+
+  // Take initial screenshot
+  await window.screenshot({ path: "tests/screenshots/intro.png" });
```

Comment:
I think for Windows compatibility it would make sense to use `path.resolve` here rather than hard-coded paths. Something like `path.resolve(__dirname, "tests", "screenshots", "intro.png")` should work, but we might have to play with the specifics a bit